styles.css

```diff
@@ -1,42 +1,102 @@
 html {
-    box-sizing:border-box;
-    margin: 0px;
+  box-sizing: border-box;
+  margin: 0;
 }
 
 body {
-    background-color: #ffffff;
-    margin: 0;
+  background-color: #fff;
+  margin: 0;
 }
 
 /* Flex Configure */
+.Nav-all {
+  width: 480px;
+}
+
+.Nav-all ul {
+  margin: 0 20px;
 
-.Nav-all ul{
-    margin: 0 auto;
-   /* border: 1px solid black; added incase of measurement */
-    list-style: none;
-    margin: 0;
-    padding: 0;
-    display: flex;
-    flex-wrap: wrap;
-    justify-content: space-between;
- 
+  /* included incase of measurement border: 1px solid black; */
+  list-style: none;
+  padding: 0;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
 }
 
+.menuNav {
+  margin-top: 23px;
+  margin-right: 5px;
+  height: 30px;
+  width: 38px;
+}
 
 .menuNav:hover {
-    background: rgba(87, 89, 53, 0.3);
-    width: fit-content;
+  background: rgba(87, 89, 53, 0.3);
+  width: fit-content;
 }
 
-.menuNav{
-    margin: 20px 20px;
-    height: 14px;
-    width: 20px;
+li .image {
+  width: 140px;
+  height: 100px;
+}
+
+/* Main Section */
 
+.My_Name {
+  margin-top: 100px;
+  width: 230px;
+  height: 100px;
+  margin-left: 37px;
+  font-family: 'Playfair Display', serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 55px;
+  line-height: 100%;
 }
 
+.all-Container {
+  align-items: center;
+  margin-top: 0;
+}
 
-li .image {
-   padding: 0;
-   width: 100%;
+.Details {
+  width: 410px;
+  height: 130px;
+  margin-left: 37px;
+  margin-top: 22px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 21px;
+  line-height: 35px;
+  letter-spacing: -0.01em;
+}
+
+#btn {
```

Comment:
[ ] kindly add the button styling: Active, Hover, and Pressed. it's in the requirements 🤓
![buttonstyle](https://user-images.githubusercontent.com/62156607/198596905-8232baf4-510d-4bf1-9574-3e2f413e3c5b.PNG)